drivers/bridge/bridge.go

```diff
@@ -98,6 +98,7 @@ type bridgeNetwork struct {
 
 type driver struct {
 	config      *configuration
+	configured  bool
```

Comment:
when working on localstore PR, I found this restriction to configure bridge driver once seems very restrictive.
I can understand that bridge driver doesnt support config changes and that is reasonable.
But the current restriction doesnt allow even new configs and hence from a code-path perspective, it becomes very difficult to consolidate all the configs in a single place and call the ConfigureNetworkDriver from docker daemon or libnetwork core. WDYT ?